include/swift/AST/DiagnosticsSema.def

```diff
@@ -8935,5 +8935,16 @@ ERROR(attr_inline_always_requires_inlinable,none,
 ERROR(attr_inline_always_no_usable_from_inline,none,
       "cannot use '@inline(always)' together with '@usableFromInline'", ())
 
+//===----------------------------------------------------------------------===//
+// MARK: Swift Performance hints
+//===----------------------------------------------------------------------===//
+
+GROUPED_WARNING(perf_hint_function_returns_array,PerfHintReturnTypeImplicitCopy,DefaultEscalate,
```

Comment:
Instead of `DefaultEscalate`, I recommend `DefaultIgnore` to make these warnings be ignored by default.